ui/src/app/applications/components/application-create-panel/application-create-setfinalizer.tsx

```diff
@@ -0,0 +1,41 @@
+import {Checkbox, HelpIcon} from 'argo-ui';
+import * as React from 'react';
+import * as ReactForm from 'react-form';
+
+export const SetFinalizerOnApplication = ReactForm.FormField((props: {fieldApi: ReactForm.FieldApi}) => {
```

Comment:
Hey Rishabh, don't want to nitpick this but the other files are named differently.  Could you rename the file to be:

set-finalizer-on-application.tsx

(since the corresponding component is SetFinalizerOnApplication)

Compare the files in application-deployment-history for example.

Reply:
Hi Rishabh, did you rename this file? I don't see the change.

Reply:
I didn't change name, will do this